src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -157,6 +157,33 @@
 				<h2 class="lender-quotes__header text-center small-12 columns">
 					What our lending <br class="so mo"> community thinks
 				</h2>
+				<div
+					v-for="lenderQuote in lenderQuotes"
+					:key="lenderQuote.name"
+					:class="`quote-card small-10 medium-8 large-5 xlarge-4 columns`"
+				>
+					<kv-responsive-image
+						class="quote-card__flourish"
+						:images="[
+							['small', lenderQuote.background],
+						]"
+						loading="lazy"
+						alt=""
+					/>
+					<img
+						:src="lenderQuote.image"
+						class="quote-card__lender-img"
```

Comment:
adding `loading="lazy"` and `:alt="lenderQuote.attribution"` would be good here.